collect_app/src/main/java/org/odk/collect/android/projects/ProjectImporter.kt

```diff
@@ -15,11 +15,19 @@ class ProjectImporter(
         metaSettings.save(MetaKeys.CURRENT_PROJECT_ID, DEMO_PROJECT_ID)
     }
 
+    // Now it does the same like importDemoProject() but it should be changed later
     fun importExistingProject() {
-        // TODO
+        val project = Project("Demo project", "D", "#3e9fcc", DEMO_PROJECT_ID)
+        projectsRepository.add(project)
+        metaSettings.save(MetaKeys.CURRENT_PROJECT_ID, DEMO_PROJECT_ID)
     }
 
     companion object {
-        const val DEMO_PROJECT_ID = "DEMO"
+        /*
+        Should be empty to easily access existed settings (general and admin) that had be saved in versions
+        prior to the one that implemented "Projects" and treat them as those which belong to
+        the imported existed project.
+         */
```

Comment:
Ah I'd thought about this differently. In my head all projects would have a prefix now that we always have a current project. That seems simpler to me, but I could be wrong?

Reply:
Hmmm maybe we did and I've just forgotten. It's already been a while. Agreed we can keep this for the PR and then we can discuss next week.